.github/workflows/main.yml

```diff
@@ -1,5 +1,9 @@
 name: Claude Code Review
 
+permissions:
```

Comment:
The changes look good overall. Moving permissions to the top level improves clarity. Using a personal token instead of GITHUB_TOKEN may provide more flexibility, but ensure it's properly secured.

``` 
permissions:
  contents: read
  pull-requests: write
  issues: write
```